hunting/llm/docs/aws_bedrock_ignore_previous_prompt_detection.md

```diff
@@ -33,7 +33,7 @@ from logs-aws_bedrock.invocation-*
 
 ## Notes
 
-- Examine flagged interactions for patterns or anomalies in user requests that may indicate malicious intent to expose LLM vulnerabilities.
+- Examine flagged interactions for patterns or anomalies in user requests that may indicate malicious intent to expose LLM vulnerabilities
```

Comment:
why did it remove the period?

Reply:
@Mikaayenson - It removed the period because the TOML file was edited, but the docs were not regenerated before the PR was merged.

Ref: https://github.com/elastic/detection-rules/blob/bb9a772870c7bc7464510ac25306f850d9c90452/hunting/llm/queries/aws_bedrock_ignore_previous_prompt_detection.toml#L29